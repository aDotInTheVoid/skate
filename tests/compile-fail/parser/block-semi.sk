fn boo() {
    if x {} else {}

    let x = 7;
    // "TODO: Allow this, see https://doc.rust-lang.org/reference/statements.html#expression-statements";
}