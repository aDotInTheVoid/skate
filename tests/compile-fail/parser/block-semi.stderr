error: Unexpected token
  ┌─ $DD/tests/compile-fail/parser/block-semi.sk:4:5
  │
4 │     let x = 7;
  │     ^^^
  │
  = Expected: "!="
  = Expected: "&&"
  = Expected: "("
  = Expected: ")"
  = Expected: "*"
  = Expected: "+"
  = Expected: ","
  = Expected: "-"
  = Expected: "."
  = Expected: "/"
  = Expected: ";"
  = Expected: "<"
  = Expected: "<="
  = Expected: "="
  = Expected: "=="
  = Expected: ">"
  = Expected: ">="
  = Expected: "["
  = Expected: "]"
  = Expected: "else"
  = Expected: "fn"
  = Expected: "{"
  = Expected: "||"
  = Expected: "}"

